### PR TITLE
AssetBundle - fix docs

### PR DESCRIPTION
**Pull request by eventhorizonpl**
Hi,

This PR updates some docs in Asset Bundle.

Best regards,
Michal

| Q | A |
| --- | --- |
| Bug fix? | no |
| New feature? | no |
| BC breaks? | no |
| Deprecations? | no |
| Tests pass? | yes |
| License | MIT |

### CODE REVIEW

**Comment by eventhorizonpl** [comment]
btw. these should be also docs for variables?

**Comment by WouterJ** [comment]
I can be completely wrong (feel free to comment if I am), but imo none of the added docs does add something. Because of the typehint, you already know the type and the description with `A <typehint> instance` tells just as much as the typehint. Same applies to `Constructor.` for a method called `__construct()`.

Don't get me wrong, I like that you are putting time into improving Symfony's code, but I'm not sure if this change any value.

File: src/Symfony/Component/Asset/Context/RequestStackContext.php [hunk 1]
```diff
@@ -22,6 +22,11 @@ class RequestStackContext implements ContextInterface
 {
     private $requestStack;
 
+    /**
+     * Constructor.
+     *
+     * @param RequestStack $requestStack A RequestStack instance
+     */
```
Comment (fabpot) [inline src/Symfony/Component/Asset/Context/RequestStackContext.php]: Should be removed.

File: src/Symfony/Component/Asset/Package.php [hunk 2]
```diff
@@ -26,6 +26,12 @@ class Package implements PackageInterface
     private $versionStrategy;
     private $context;
 
+    /**
+     * Constructor.
+     *
+     * @param VersionStrategyInterface $versionStrategy The version strategy
+     * @param ContextInterface         $context         The context
+     */
```
Comment (fabpot) [inline src/Symfony/Component/Asset/Package.php]: Should be removed.

File: src/Symfony/Component/Asset/Package.php [hunk 3]
```diff
@@ -53,6 +59,8 @@ public function getUrl($path)
     }
 
     /**
+     * Returns context.
+     *
```
Comment (fabpot) [inline src/Symfony/Component/Asset/Package.php]: Should be removed.

File: src/Symfony/Component/Asset/Package.php [hunk 4]
```diff
@@ -60,11 +68,23 @@ protected function getContext()
         return $this->context;
     }
 
+    /**
+     * Returns version strategy.
+     *
+     * @return VersionStrategyInterface
+     */
```
Comment (fabpot) [inline src/Symfony/Component/Asset/Package.php]: Only the `@return` line should be kept

File: src/Symfony/Component/Asset/Package.php [hunk 5]
```diff
@@ -60,11 +68,23 @@ protected function getContext()
...
         return $this->versionStrategy;
     }
 
+    /**
+     * Determinates if url is absolute.
+     *
+     * @param string $url An url
+     *
+     * @return bool
+     */
```
Comment (fabpot) [inline src/Symfony/Component/Asset/Package.php]: Should be removed.

File: src/Symfony/Component/Asset/PathPackage.php [hunk 6]
```diff
@@ -29,8 +29,11 @@ class PathPackage extends Package
     private $basePath;
 
     /**
+     * Constructor.
+     *
      * @param string                   $basePath        The base path to be prepended to relative paths
      * @param VersionStrategyInterface $versionStrategy The version strategy
+     * @param ContextInterface         $context         The context
      */
     public function __construct($basePath, VersionStrategyInterface $versionStrategy, ContextInterface $context = null)
```
Comment (fabpot) [inline src/Symfony/Component/Asset/PathPackage.php]: Should be removed.

File: src/Symfony/Component/Asset/UrlPackage.php [hunk 7]
```diff
@@ -39,8 +39,11 @@ class UrlPackage extends Package
     private $sslPackage;
 
     /**
+     * Constructor.
+     *
      * @param string|array             $baseUrls        Base asset URLs
      * @param VersionStrategyInterface $versionStrategy The version strategy
+     * @param ContextInterface         $context         The context
```
Comment (fabpot) [inline src/Symfony/Component/Asset/UrlPackage.php]: Should be removed.

File: src/Symfony/Component/Asset/UrlPackage.php [hunk 8]
```diff
@@ -118,6 +121,15 @@ protected function chooseBaseUrl($path)
...
 
+    /**
+     * Returns ssl urls.
+     *
+     * @param array $urls An array of urls.
+     *
+     * @return array An array of ssl urls.
+     *
+     * @throws InvalidArgumentException If one of urls is not valid.
+     */
```
Comment (fabpot) [inline src/Symfony/Component/Asset/UrlPackage.php]: we do not document private methods.

File: src/Symfony/Component/Asset/VersionStrategy/StaticVersionStrategy.php [hunk 9]
```diff
@@ -21,6 +21,12 @@ class StaticVersionStrategy implements VersionStrategyInterface
     private $version;
     private $format;
 
+    /**
+     * Constructor.
+     *
+     * @param string $version Version number
+     * @param string $format  Url format
+     */
     public function __construct($version, $format = null)
```
Comment (fabpot) [inline src/Symfony/Component/Asset/VersionStrategy/StaticVersionStrategy.php]: "Constructor" line should be removed.

**Comment by fabpot** [comment]
Thank you @eventhorizonpl.

**Pull request closed without merging**